src/tea_json.ml

```diff
@@ -168,15 +168,15 @@ module Decoder = struct
       )
 
   let at fields dec =
-    List.fold_right field fields dec
+    Belt.List.reduceReverse fields dec field
```

Comment:
What on earth is `reduceReverse`?  I've never heard that name in any language in 30 years either.  Can a more descriptive/appropriate name be used?

Reply:
Looks like it's going to be renamed to `reduceRight`: https://github.com/BuckleScript/bucklescript/pull/2631.

Reply:
Ah, so it doesn't reverse the output, it is just a right fold (reversed input), that makes more sense than it being called 'reduceReverse'.  ^.^;

What is with all these very non-standard names in Belt I'm curious?  o.O?